tfhe/src/shortint/engine/server_side/scalar_bitwise_op.rs

```diff
@@ -30,7 +30,7 @@ impl ShortintEngine {
     pub(crate) fn smart_scalar_bitand(
         &mut self,
         server_key: &ServerKey,
-        lhs: &mut Ciphertext,
+        lhs: &Ciphertext,
```

Comment:
needs to be mutable

Reply:
It seems to me that this operations never modifies the input and always applies a LUT
Which condition do you suggest we modify this input under?

Reply:
same for the other ops of the module

Reply:
ah this is tricky should we clean the inputs if we are just applying a lut

Reply:
as we never need to clean it in that case, no should be fine I guess in this case my bad